executor/executor_test.go

```diff
@@ -8569,7 +8569,7 @@ func (s testSerialSuite) assertTemporaryTableNoNetwork(c *C, temporaryTableType
 	// With that failpoint, all requests to the TiKV is discard.
 	rs, err := tk1.Exec("select * from normal")
 	c.Assert(err, IsNil)
-	blocked := make(chan struct{})
+	blocked := make(chan struct{}, 1)
```

Comment:
It should be a buffer channel to unblocking L8576, otherwise the anon goroutine blocked and leaks.